server/eclipse-project/src/main/java/us/freeandfair/corla/query/BallotManifestInfoQueries.java

```diff
@@ -182,6 +189,27 @@ Find the batch(bmi) that would hold the sequence number given.
     return result.stream().findFirst();
   }
 
+  /**
+   * Get the max sequence number which is the total number of CVRs there should be
+   */
+  public static OptionalLong maxSequence(final Long the_county) {
+    OptionalLong result = OptionalLong.empty();
+
+    try {
+      final Session s = Persistence.currentSession();
+      final CriteriaBuilder cb = s.getCriteriaBuilder();
+      final CriteriaQuery<Long> cq = cb.createQuery(Long.class);
+      final Root<BallotManifestInfo> root = cq.from(BallotManifestInfo.class);
+      cq.select(cb.max(root.get("my_sequence_end")));
+      cq.where(cb.equal(root.get("my_county_id"), the_county));
+
+      final TypedQuery<Long> query = s.createQuery(cq);
+      result = OptionalLong.of(query.getSingleResult());
+    } catch (final PersistenceException e) {
+      // ignore
```

Comment:
I'd rather see some logging than a silent catch, even if it is `log.error("this is never going to happen");`